### PR TITLE
Add authentication.

### PR DESCRIPTION
**Pull request by clemenshelm**

### CODE REVIEW

File: server/startup.coffee [hunk 1]
```diff
@@ -9,6 +9,11 @@ Meteor.startup ->
       [..., suffix] = file.name.split '.'
       "#{metaContext.id}.#{suffix}"
 
+  # make sure there's an admin
+  if Meteor.users.find({}).count() == 0
+    Accounts.createUser email: 'clemens.helm@gmail.com', password: 'change now!'
```
Comment (DominikGuzei) [inline server/startup.coffee]: You can leave it like this now, but I would suggest to move these things into migrations later on. This makes it easier to keep all seed-data and changes to existing data in one place. This package is quite ok for that: https://github.com/percolatestudio/meteor-migrations

Comment (clemenshelm) [inline server/startup.coffee]: Cool, thanks for the input! I really value your feedback! I'll check it out now, because I also don't like it here.

**Comment by clemenshelm** [comment]
I updated it with migrations. I don't like the migration package very much, as it only really works for one developer. When there's a team and many people create migrations, there will be a mess, because several people might create migrations with the same versions or newer versions might get merged before older versions. The package can't cope with these situations.

Anyway, at the moment it's only me coding on the project, so no problem so far ;)

**Comment by DominikGuzei** [comment]
Sure, you just have to use timestamps as the version number :wink: I also use it for Prisma and it works quite well

**Comment by DominikGuzei** [comment]
If you need the current timestamp, the browser console is your friend: `Date.now()`

**Comment by clemenshelm** [comment]
This solves the collision problem, which is the minor problem. But let's say I open two pull requests, each containing a migration. When you merge the pull request with the newer migration first and the other one afterwards, the migration will never get run, because it will be considered "already run" by the package, since it has a lower version number.

**Comment by DominikGuzei** [comment]
good point :wink: How does rails solve that problem?

**Comment by clemenshelm** [comment]
It stores all (timestamps of) migrations that were already run in a database table. Then it only runs those which aren't in the table yet.

**Comment by DominikGuzei** [comment]
Ok, should be a pretty simple patch to migrations package … maybe we could submit that someday :wink:

**Comment by clemenshelm** [comment]
Definitely ;)

So how about this Pull Request? Feedback or merge?

**Comment by DominikGuzei** [comment]
merge! :+1:

**Pull request merged**